### ISSUE TITLE
Tornado websockets event Handlers registration are incorrect

### ISSUE THREAD

**Issue by mavenAtHouzz** [saltstack/salt#22605]
**Target module:** ``netapi/rest_tornado/__init__.py``

The token pattern at line 60 is incorrect. The {0} format will just put the used hashing length after the \w regex which is actually should be meant for \w length.

**Intended pattern (for md5 32bit example):** [0-9A-Fa-f]{32}
**Actual pattern after format:** [0-9A-Fa-f]32

And following is a tested patch:

```diff
--- netapi/rest_tornado/__init__.py     2015-02-12 09:55:39.000000000 -0800
+++ netapi/rest_tornado/__init__.new.py    2015-04-09 14:06:47.482869347 -0700
@@ -56,7 +56,8 @@
     if mod_opts.get('websockets', False):
         from . import saltnado_websockets

-        token_pattern = r"([0-9A-Fa-f]{0})".format(len(getattr(hashlib, __opts__.get('hash_type', 'md5'))().hexdigest()))
+        token_pattern = r"([0-9A-Fa-f]{{{0}}})".format(len(getattr(hashlib, __opts__.get('hash_type', 'md5'))().hexdigest()))
         all_events_pattern = r"/all_events/{0}".format(token_pattern)
         formatted_events_pattern = r"/formatted_events/{0}".format(token_pattern)
         logger.debug("All events URL pattern is {0}".format(all_events_pattern))
```

**Comment by basepi** [comment]
Good catch! Would you mind submitting a pull request?

**Comment by rallytime** [comment]
Fixed in #23261
